src/test/java/com/epam/tc/hw3/ex2/DifferentElementsPageData.java

```diff
@@ -0,0 +1,20 @@
+package com.epam.tc.hw3.ex2;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class DifferentElementsPageData {
+
+    public static String PROPERTIES_PATH = "src/test/resources/hw3/test.properties";
```

Comment:
Can be moved in property file. Code duplicate from IndexPageData

Reply:
Property was removed